packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -233,6 +242,7 @@ export class SerializedStateManager {
 					pendingRuntimeState,
 					baseSnapshot: this.snapshot.baseSnapshot,
 					snapshotBlobs: this.snapshot.snapshotBlobs,
+					snapshotSequenceNumber: this.snapshot.snapshotSequenceNumber,
```

Comment:
i don't think we should modify the serialized format for this. since we only need this information for refresh, just get the attributes from this.snapshot within the this.refreshSnapshot promise

Reply:
@tyler-cai-microsoft  is about to do that change so I thought I could just reuse some of his changes se he can deal with fewer merge conflicts

Reply:
modifying our serialized format is a big deal, and generally should have a full design review, so finding other ways to do things is generally preferred.